gradle.properties

```diff
@@ -11,6 +11,9 @@ databasePassword=password
 tomcatServerPort=8080
 
 sslEnabled=false
+salt=23C1C508F5250AE3
```

Comment:
I think those names should be more meaningful and be in a separate group (it have nothing to do with the https certificate)
it can be: encryptionSalt, encryptionKey, encryptionIv